project-addons/scheduled_shipment/views/sale_view.xml

```diff
@@ -22,7 +22,8 @@
             <field name="arch" type="xml">
                 <xpath expr="//header" position="inside">
                     <!-- TODO: revisar estados donde aparece el boton -->
-                    <button name="action_shedule" states="confirmed,waiting,partially_available,assigned" string="Schedule Shipping" type="object" groups="base.group_user"/>
+                    <button name="action_shedule" string="Schedule Shipping" type="object" groups="base.group_user" 
+                        attrs="{'invisible':['|', ('picking_type_id','!=',2), '|', ('state','in',('assigned')), ('block_picking','=',True)]}"/>
```

Comment:
podrías usar picking_type_code != 'outgoing' y así no usas un id interno en código

Reply:
Hola Omar! 
He probado con:
`"{'invisible':['|', ('picking_type_id','!=',2), '|', ('state','in',('assigned')), ('block_picking','=',True), '&amp;',('picking_type_code','!=','outgoing')]}"`

y con:
`"{'invisible':['|', ('picking_type_id','!=',2), '|', ('state','in',('assigned')), ('block_picking','=',True), '|;',('picking_type_code','!=','outgoing')]}"`

pero me aparece en todos los casos y no cuando el usuario es del grupo de administración. No sé a qué te refieres con utilizar eso...

Reply:
Ya lo he cambiado en el úlitmo commit @omar7r